README.md

```diff
@@ -17,31 +17,80 @@ Overview
 
 This dockerfile has been created to facilitate the creation of
 [ITK](https://github.com/InsightSoftwareConsortium/ITK)
-[Doxygen](https://doxygen.nl/) documentation in a reproducible way.
+[Doxygen](https://doxygen.nl/)
+
+documentation in a reproducible way.
```

Comment:
Why is the above empty line added?

Reply:
I used an empty line to limit the character length of each line, but this made each line its paragraph. Instead, I've committed changes that use the `<br>` tags to add line breaks. I separated the section from the original documentation using this 79-character width line as well.

Let me know if the new commit solves this issue!

Reply:
The solution with `<br>` worked in Markdown preview but didn't render correctly on Github itself. I tried to add a double space at the end of each line and this worked the intended way. 

Sorry for the confusion,
@viv511

Reply:
Please, do not add spaces/double spaces at the end of lines; when you hit 79 chars, just start a new line, and that is enough.